### ISSUE TITLE
o-forms: radio input labels are not are not available to assistive technologies - ID: DAC_Hidden_Labels_01

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#531]
# Issue
This is from page 88 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=inverse-form&system_code=origami-repo-data&brand=master>

The labels for the radio buttons have been hidden using the aria-hidden attribute. This means that users of assistive technologies are unable to identify and interact with the buttons correctly.

 
# Proposed Solution:
Remove the aria-hidden attribute from the labels and use explicit labels for the radio buttons with matching ‘for’ and ‘id’ values. This will enable users of assistive technologies to identify and interact with the radios correctly.

## Example:
```html
<div class="o-forms-input--radio-box__container">
	<label for="radio1">Yes</label>
	<input id="radio1" type="radio" name="box">
	<label for="radio1">No</label>
	<input id="radio1" type="radio" name="box" checked="">
</div>
```

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=interactive&system_code=origami-repo-data&brand=master> and found on page 159 of the audit report
ID: DAC_Hidden_Labels_02

**Comment by notlee** [comment]
this is fixed https://github.com/Financial-Times/origami/commit/034f1f755ad6ea7f6ff2adc7df397c0bde050e40#diff-df8529719e0d60ef851d27d8aaa8e71a5a658d75541d0179dd7af9c574a774b5R54
